sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,386 @@
+  - Added complex support for the reduce and scan group algorithms. [90a4dc7]
+  - Added support for SYCL 2020 methods in the group class.  [73d59ce]
+  - Added `SYCL_RT_WARNING_LEVEL` env variable which allows to control amount of warnings and performance hints the runtime library may print. [2741010]
+  - Added `tanh` (for floats/halfs) and `exp2` (for halfs) native definitions for CUDA backend. [250c498]
+  - Added bf16 builtins for fma, fmin, fmax and fmax on CUDA backend. [62651dd]
+  - Added support for USM buffer location properties which allows to specify at what memory location the device usm allocation should be in. [12c988a]
+  - Added support for buffer_location property to the sycl::buffer. [9808525]
+  - Added single_task support for ESIMD_EMU backend. [2331160]
+  - Added support for SVM 1,2,4-elements gather/scatter for ESIMD. [e200720]
+  - Added support for bf16 builtins operating on storage types for CUDA backend. [413a9ef]
+  - Added support for backend_version device property for CUDA backend. [4b1a4bc]
+  - Added support for round-robin submissions to multiple compute CCS for the L0 backend. Disabled by default, can be controlled using `SYCL_PI_LEVEL_ZERO_USE_COMPUTE_ENGINE`. [a836c87]
+  - Added support for buffer migration for contexts with multiple devices in the L0 plugin. [7baf152]
+  - Added mode where the L0 plugin uses immediate commandlists instead of standard commandlists. This mode is disabled by default, can be enabled using `SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS` env variable. [b9cb1d1]
+  - Added support for `sycl::get_native(sycl::buffer)` for OpenCL and CUDA backends. [8b3c8c4]
+  - Added reduction overloads accepting `span`. [863383b]
+  - Added LSC support for ESIMD_EMULATOR backend. [b78bf00]
+  - Added `half` type support for `__esimd_convertvector_to/from`. [0bfffd6]
+  - Added `buffer_allocator` SYCL 2020 conformant variant. [53430c8]
+  - Added support for the USM buffer location property in `malloc_shared`. [6e89821] [9f61c8e][8c4d9a5]
+  - Added support for the USM buffer location property in `malloc_host`. [2c7caab]
+  - Added experimental context and device interoperability support for CUDA. [f0df89a]
+  - Added support for memory intrinsics for the ESIMD_EMULATOR plugin. [1a8f501]
+  - Added support for named barrier APIs for ESIMD. [1df0038]
+  - Added support for dpas API for ESIMD. [5881938]
+  - Added support for lsc memory access APIs for ESIMD. [4bd50e7]
+  - Added support for `buffer_location` property to the `sycl::buffer`. [9808525]
+  - Added `single_task` support for ESIMD_EMULATOR backend. [2331160]
+  - Added support for SVM 1,2,4-elements gather/scatter for ESIMD. [e200720]
+  - Added support for `bf16` builtins operating on storage types for CUDA backend. [413a9ef]
+  - Added `backend_version` device property. [4b1a4bc]
+  - Added support for the `invoke_simd` feature. [4072557][8471ff3][8c7bb45][62afb59][3e1c1bf][8c7bb45]
```

Comment:
[8c7bb45] is repeated twice. Also, this seems like a better fit in 'Tools' section.

Reply:
Removed duplicate. Commits implementing this feature touch many parts of the compiler - frontend, llvm passes, sycl-post-link tool etc. That's why I put it in SYCL Compiler section. Correct me if my understanding is wrong.